nvme.go

```diff
@@ -54,6 +54,12 @@ const (
 
 	// NVMePortDefault - NVMe TCP port
 	NVMePortDefault = ":4420"
+
+	// PowerMaxOUIPrefix - PowerMax format 6 OUI prefix
+	PowerMaxOUIPrefix = "6000097"
```

Comment:
Hi @donatwork  I do not know much about this. Does this value remain unique across all PowerMax arrays of all versions?